cmd/tpcc/main.go

```diff
@@ -32,11 +35,14 @@ import (
 
 var (
 	clientCount  = flag.Int("client", 5, "client count")
-	requestCount = flag.Int("request-count", 1000, "client test request count")
+	requestCount = flag.Int("request-count", math.MaxInt64, "client test request count")
```

Comment:
Why?

Reply:
In this case, I use `runTime` to control the total run time of each round, so I adjust the default value to a very big number to prevent each round finishes too quickly.